packages/@aws-cdk/aws-lambda/test/integ.autoscaling.ts

```diff
@@ -0,0 +1,43 @@
+import * as appscaling from '@aws-cdk/aws-applicationautoscaling';
+import * as cdk from '@aws-cdk/core';
+import * as lambda from '../lib';
+
+/**
+ * Stack verification steps:
+ * aws application-autoscaling describe-scalable-targets --service-namespace lambda --resource-ids function:aws-lambda-autoscaling-<lambda name>:prod
```

Comment:
Nit: I like to add the lambda name as a `CfnOutput` so that it's printed in the output of `cdk deploy` and can be easily plugged into this command.